.zenodo.json

```diff
@@ -170,6 +184,9 @@
     {
       "name": "St-Jean, Samuel"
     },
+    {
+      "name": "Panfilov, Egor"
```

Comment:
@soupault Please let me know if you would like your name presented differently, and if you have an affiliation/ORCID that you want associated with your contributions.

This will be how your name is presented as an author on the [Zenodo archive](https://doi.org/10.5281/zenodo.591597). If you prefer not to be an author, let me know.

Reply:
@effigies just ORCID would do, thank you. It's 0000-0002-2500-6375 .